security-analysis/src/main/java/eu/itesla_project/security/SecurityAnalysisResult.java

```diff
@@ -18,7 +20,8 @@
 
     private final List<PostContingencyResult> postContingencyResults;
 
-    public SecurityAnalysisResult(PreContingencyResult preContingencyResult, List<PostContingencyResult> postContingencyResults) {
+    public SecurityAnalysisResult(@JsonProperty("preContingencyResult") PreContingencyResult preContingencyResult, 
```

Comment:
We have to add a version parameter to SecurityAnalysisResult export to be able, in the future, to manage several versions of this export.